llama_hub/llama_packs/panel_chatbot/base.py

```diff
@@ -0,0 +1,42 @@
+"""Provides the PanelChatPack"""
+from typing import Dict, Any
+import os
+from llama_index.llama_pack.base import BaseLlamaPack
+
+ENVIRONMENT_VARIABLES = [
+    "GITHUB_TOKEN",
+    "OPENAI_API_KEY",
+]
+
+
+class PanelChatPack(BaseLlamaPack):
+    """Panel chatbot pack."""
+
+    def get_modules(self) -> Dict[str, Any]:
+        """Get modules."""
+        return {}
+
+    def run(self, *args: Any, **kwargs: Any) -> Any:
+        """Run the pipeline."""
+        for variable in ENVIRONMENT_VARIABLES:
+            if variable not in os.environ:
+                raise ValueError(f"{variable} is not set")
```

Comment:
Unfortunately this is causing our unit test `test_llama_packs_library_matches` to fail. Can we disable this for now? We gotta think of a good way to incorporate this and related checks into our unit tests.

Reply:
Done